web/mediator/manifest.js

```diff
@@ -20,8 +20,43 @@ export async function getWebAppManifest({origin, timeout = 1000}) {
 }
 
 async function _getWebAppManifest({origin}) {
-  const url = `${origin}/manifest.json`;
+  // urls are in order of priority
+  const urls = [
+    `${origin}/manifest.json`,
+    `${origin}/manifest.webmanifest`
+  ];
+  const responses = await Promise.allSettled(
+    urls.map(url => _fetchWithCache(url))
+  );
```

Comment:
I think this can be shortened to:

```suggestion
  const responses = await Promise.allSettled(urls.map(_fetchWithCache));
```

Reply:
Definitely, not sure why I left this - thanks!